package.json

```diff
@@ -29,7 +29,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git@github.com:briza-insurance/illogical.git"
+    "url": "git+ssh://git@github.com/briza-insurance/illogical.git"
```

Comment:
does the new npm version require this change?

Reply:
Don't think so. NPM already made this change before publishing